src/transformers/models/autoformer/modeling_autoformer.py

```diff
@@ -1869,12 +1868,31 @@ def forward(
         ...     past_time_features=batch["past_time_features"],
         ...     past_observed_mask=batch["past_observed_mask"],
         ...     static_categorical_features=batch["static_categorical_features"],
-        ...     static_real_features=batch["static_real_features"],
         ...     future_time_features=batch["future_time_features"],
         ... )
 
         >>> mean_prediction = outputs.sequences.mean(dim=1)
-        ```"""
+        ```
+
+        <Tip>
+
+        The AutoformerForPrediction can also use static_real_features. To do so, set num_static_real_features to 1
```

Comment:
This example should show how / where to set `num_static_real_features`, and explain why it's 1

Reply:
Sure, added setting this value in model configuration